models/src/test/scala/coop/rchain/models/rholang/SortTest.scala

```diff
@@ -500,6 +504,77 @@ class ParSortMatcherSpec extends FlatSpec with Matchers {
     result.term should be(sortedParMatch)
   }
 
+  it should "sort Lets based on declarations and then body" in {
```

Comment:
Looks to me like sorting on the body is missing

Reply:
You want a separate test that sorts some number of `let` statements with the same declarations but different bodies?

Reply:
No, I think adding one more let here with a declaration equal to some other let but with a different body in a wrong position should be enough